compiler/rustc_feature/src/active.rs

```diff
@@ -644,6 +644,9 @@ declare_features! (
     /// Allows associated types in inherent impls.
     (active, inherent_associated_types, "1.52.0", Some(8995), None),
 
+    /// Adds the `bool::not` method so that you don't need `core::ops::Not` imported.
+    (active, bool_not_method, "1.52.0", None, None),
```

Comment:
Are you sure this needs a feature gate in the compiler itself? I would expect `#[unstable]` to work with only changes to the standard library.

Reply:
I'm not sure. I'm just going by the page of the rustc-dev-guide that you linked me to >_>

Reply:
Hmm, I don't see it in https://rustc-dev-guide.rust-lang.org/stability.html#unstable. https://rustc-dev-guide.rust-lang.org/implementing_new_features.html#stability-in-code is for compiler features, not library features.

Reply:
seems to be working so far when i took it out.